src/test/java/plugins/LdapPluginTest.java

```diff
@@ -32,7 +32,7 @@
  */
 @Category(DockerTest.class)
 @WithDocker
-@WithPlugins("ldap@1.10")
+@WithPlugins("ldap@1.15")
```

Comment:
This effectively removes ATH coverage from LDAP plugin for all older version than the latest - released an hour ago. Only the test methods that really need it should require it.